user-guide/Advanced_Modules/Correlation/Tutorials/Correlation_Tutorial_Element_Timeout.md

```diff
@@ -0,0 +1,102 @@
+
+1. Add an alarm filter that filters on alarms from the element "Network Switch".
+
+   See [filtering and grouping base alarms for Correlation rules](xref:Filtering_and_grouping_base_alarms_for_Correlation_rules) for more information.
+
+   ![Adding an alarm filter](~/user-guide/images/Correlation_Adding_Alarmfilters.png)
+
+1. Add a rule condition that triggers when the severity is equal to "Timeout".
+
+   See [adding rule conditions in Correlation rules](xref:Adding_rule_conditions_in_Correlation_rules) for more information.
+
+   ![Adding a rule condition](~/user-guide/images/Correlation_Adding_RuleConditions.png)
+
+1. Select the "Persistent event" trigger mechanism and fill in 5 minutes.
+
+   The Correlation Rule will wait until the rule conditions are met for 5 minutes before acting and executing its actions.
+
+   See [adding rule conditions in Correlation rules](xref:Adding_rule_conditions_in_Correlation_rules) for more information.
+
+   ![Configuring *persistent event time*](~/user-guide/images/Correlation_PersistentEvent.png)
+
+1. Add the RunScript action and select the "Generate Information Event (network switch timed out)" script.
+
+   This Automation script will generate an information event when executed. Note that these scripts also can do other things.
+
+   See [adding a RunScript action](xref:Running_an_Automation_script_from_a_Correlation_rule) for more information.
+
+   ![Adding the *run script* action](~/user-guide/images/Correlation_Add_Run_Script_Action.png)
+
+1. Optionally, add the Send email action.
+
+   See [sending an email](xref:Sending_an_email) for more information on how to do this.
+
+   ![Adding the *send mail* action](~/user-guide/images/Correlation_Add_Send_Mail_Action.png)
+
+   > [!NOTE]
+   > This action will only work if a mail server has been configured. See [Configuring outgoing email](xref:Configuring_outgoing_email)
+
+1. Optionally, add other rule actions according to your preference.
+
+   For more information on how to add rule actions, go to [adding rule actions in Correlation rules](xref:Adding_rule_actions_in_Correlation_rules).
+
+> [!NOTE]
+> Make sure to have a look at some [best practices when making Correlation Rules](xref:Best_Practices_When_Creating_Correlation_Rules). Correlation is a powerful tool, but can have a negative impact on your system if set up the wrong way.
+
+## Step 3: Generate a timeout alarm
+
+1. In the Surveyor, select the *Network Switch* element.
+
+1. Click the *Generate Timeout* button and wait 30 seconds (i.e. the default duration DataMiner waits when an element does not respond before generating a timeout alarm).
+
+   A timeout alarm will then be generated.
+
+## Step 4: Verify whether the rule actions have been executed
+
+1. Wait for five minutes after the alarm has gone into timeout.
+
+1. Check whether an information event has been generated in the Alarm Console.
```

Comment:
Probably change the text 'Correlation KATA: Network Switch timed out for more than 1h (Script 'Correlation KATA switch timeout Script')' (generated by the script) to 'more than 5 minutes'